tests/lib/resource_test.py

```diff
@@ -301,7 +313,10 @@ def test_test_entrypoint_success():
     mock_model._deserialize.side_effect = [None, None]
 
     resource = Resource(mock_model)
-    mock_handler = resource.handler(Action.CREATE)(Mock(return_value=sentinel.response))
+    progress_event = Mock(
+        "cloudformation_cli_python_lib.interface.ProgressEvent", autospec=True
+    )()
```

Comment:
same comment as John had before, what's the benefit of mocking this?

Reply:
oops, thought I'd already changed all of these